lib/heroku/command/pg_backups.rb

```diff
@@ -26,9 +26,9 @@ def copy
     message = "WARNING: Destructive Action"
     message << "\nThis command will remove all data from #{target.name}"
     message << "\nData from #{source.name} will then be transferred to #{target.name}"
-    message << "\nThis command will affect the app: #{app}"
+    message << "\nThis command will affect the app: #{attachment.app}"
 
-    if confirm_command(app, message)
+    if confirm_command(attachment.app, message)
```

Comment:
Good call--if you're restoring *from* a Heroku Postgres database to a non-HPG DB, only `name` and `url` fields of the `target` will be valid (it's a `MaybeAttachment`; see below in `resolve_db_or_url`). The